src/limbo/opt/nlopt.hpp

```diff
@@ -0,0 +1,61 @@
+#ifndef LIMBO_OPT_NLOPT_HPP
+#define LIMBO_OPT_NLOPT_HPP
+
+#ifndef USE_NLOPT
+#warning No NLOpt
+#else
+#include <Eigen/Core>
+
+#include <vector>
+
+#include <nlopt.hpp>
+
+namespace limbo {
+    namespace opt {
+        template <typename Params, nlopt::algorithm Algorithm = nlopt::LD_MMA>
```

Comment:
Only because it was the default in their examples. No other reason. If you have any other preference, please do not hesitate. You can see the NLOpt algorithms [here](http://ab-initio.mit.edu/wiki/index.php/NLopt_Algorithms).